docs/cita/configuration/chain-config.md

```diff
@@ -29,8 +29,8 @@ usage: create_cita_config.py create [-h]
                                     [--resource_dir RESOURCE_DIR]
                                     [--grpc_port GRPC_PORT]
                                     [--jsonrpc_port JSONRPC_PORT]
-                                    [--ws_port WS_PORT]
-                                    [--enable_tls]
+                                    [--ws_port WS_PORT] [--enable_tls]
```

Comment:
One line?

Reply:
Run the command:
```shell
./bin/cita create --help
```
and will get the output like this:
```
usage: create_cita_config.py create [-h]
                                    [--authorities AUTHORITY[,AUTHORITY[,AUTHORITY[,AUTHORITY[, ...]]]]]
                                    [--chain_name CHAIN_NAME]
                                    [--nodes IP:PORT[,IP:PORT[,IP:PORT[,IP:PORT[, ...]]]]]
                                    [--super_admin SUPER_ADMIN]
                                    [--contract_arguments Contract.Argument=Value [Contract.Argument=Value ...]]
                                    [--timestamp TIMESTAMP]
                                    [--resource_dir RESOURCE_DIR]
                                    [--grpc_port GRPC_PORT]
                                    [--jsonrpc_port JSONRPC_PORT]
                                    [--ws_port WS_PORT] [--enable_tls]
                                    [--enable_version] [--stdout]

optional arguments:
  -h, --help            show this help message and exit
  --authorities AUTHORITY[,AUTHORITY[,AUTHORITY[,AUTHORITY[, ...]]]]
                        Authorities (addresses) list.
  --chain_name CHAIN_NAME
                        Name of the new chain.
  --nodes IP:PORT[,IP:PORT[,IP:PORT[,IP:PORT[, ...]]]]
                        Node network addresses for new nodes.
  --super_admin SUPER_ADMIN
                        Address of super admin.
  --contract_arguments Contract.Argument=Value [Contract.Argument=Value ...]
                        Update constructor arguments for system contract. Can
                        be specify more than once.
  --timestamp TIMESTAMP
                        Specify a timestamp to use.
  --resource_dir RESOURCE_DIR
                        Chain resource directory.
  --grpc_port GRPC_PORT
                        grpc port for this chain
  --jsonrpc_port JSONRPC_PORT
                        jsonrpc port for this chain
  --ws_port WS_PORT     websocket port for this chain
  --enable_tls          The data is encrypted and transmitted on the network
  --enable_version      Jsonrpc will return cita version
  --stdout              Logs will output to stdout
```

Reply:
Need to fix the script of CITA.

should be:

```
usage: create_cita_config.py create [-h]
                                    [--authorities AUTHORITY[,AUTHORITY[,AUTHORITY[,AUTHORITY[, ...]]]]]
                                    [--chain_name CHAIN_NAME]
                                    [--nodes IP:PORT[,IP:PORT[,IP:PORT[,IP:PORT[, ...]]]]]
                                    [--super_admin SUPER_ADMIN]
                                    [--contract_arguments Contract.Argument=Value [Contract.Argument=Value ...]]
                                    [--timestamp TIMESTAMP]
                                    [--resource_dir RESOURCE_DIR]
                                    [--grpc_port GRPC_PORT]
                                    [--jsonrpc_port JSONRPC_PORT]
                                    [--ws_port WS_PORT]
                                    [--enable_tls]
                                    [--enable_version]
                                    [--stdout]
```

Reply:
Okay!

Reply:
I see. leave it.